multicore_runtime/cloud_logging_test.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2015 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import json
+import logging
+import mock
+import os
+import unittest
+
+import cloud_logging
```

Comment:
I feel that the full path to the module as installed on a user's system should not be tightly coupled to the code in order to maintain separation of concerns. Also, using relative paths means the tests can be run from the repo in-place without the package being compiled and installed.

However, I agree that we should be using explicitly relative imports in order to avoid any ambiguity.

Using (explicitly) relative imports is consistent with the standard in other well-designed open-source libraries such as requests https://github.com/kennethreitz/requests